spotlight/evaluation.py

```diff
@@ -96,6 +96,60 @@ def sequence_mrr_score(model, test):
     return np.array(mrrs)
 
 
+def precision_recall_at_k(model, test, k):
+    """
+    Compute Precision@k and Recall@k scores. One score
+    is given for every user with interactions in the test
+    set, representing the Precision@k and Recall@k of all their
+    test items.
+
+    Parameters
+    ----------
+
+    model: fitted instance of a recommender model
+        The model to evaluate.
+    test: :class:`spotlight.interactions.Interactions`
+        Test interactions.
+    k: int or array of int,
+        The maximum number of predicted items
+    Returns
+    -------
+
+    (Precision@k, Recall@k): numpy array of shape (num_users,)
+        A tuple of Precisions@k and Recalls@k for each user in test.
```

Comment:
We should be clearer about what this returns. I suggest tuple of lists of arrays (or tuple of arrays in the case of scalar `k`), so that the i-th element of list is the `(num_users ,)` array of precision/recall at `k[i]`.